vendor.conf

```diff
@@ -0,0 +1,131 @@
+# the following lines are in sorted order, FYI
+github.com/Azure/go-ansiterm 388960b655244e76e24c75f48631564eaefade62
+github.com/Microsoft/hcsshim v0.5.2
```

Comment:
Would this transition be a good opportunity to require hashes for each vendored repo? Relying on upstream tags kind of worries me.